src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/LogsController.php

```diff
@@ -83,22 +84,16 @@ public function indexAction(LogsFilters $filters)
      */
     public function searchAction(Request $request)
     {
-        $definitionFactory = $this->get('prestashop.core.grid.definition.factory.logs');
-        $logsDefinition = $definitionFactory->getDefinition();
-
-        $gridFilterFormFactory = $this->get('prestashop.core.grid.filter.form_factory');
-        $searchParametersForm = $gridFilterFormFactory->create($logsDefinition);
-
-        $searchParametersForm->handleRequest($request);
-        $filters = [];
-
         $this->dispatchHook('actionAdminLogsControllerPostProcessBefore', ['controller' => $this]);
 
-        if ($searchParametersForm->isSubmitted()) {
-            $filters = $searchParametersForm->getData();
-        }
+        $responseBuilder = $this->get('prestashop.bundle.grid.response_builder');
```

Comment:
Why modify this controller and not use the common search one instead? This could simply be deprecated and not modified right?

Reply:
By the way the deprecate comment is missing

Reply:
Because there is a dispatchHook in this case.

Reply:
Oh I get it, so you didn't need to modify the routing either and it uses this specific search controller All good! my bad